Kudu.Services/Diagnostics/ProfileManager.cs

```diff
@@ -35,6 +35,16 @@ internal static class ProfileManager
 
         private static string _processName = System.Environment.ExpandEnvironmentVariables("%SystemDrive%\\msvsmon\\profiler\\VSStandardCollector.Dev14.exe");
 
+        static ProfileManager()
+        {
```

Comment:
Please combine this logic with private static readonly TimeSpan _profilingIisTimeout = TimeSpan.FromMinutes(3);
and remove this static assignment.